contracts/Core/StakeManager.sol

```diff
@@ -296,9 +297,10 @@ contract StakeManager is Initializable, StakeStorage, StateManager, Pause, Stake
     function unstake(uint32 stakerId, uint256 sAmount) external initialized whenNotPaused {
         require(sAmount > 0, "Non-Positive Amount");
         require(stakerId != 0, "staker.id = 0");
+        // slither-disable-next-line timestamp
```

Comment:
hmm how stakers stake is related to timestamp 
slither is suggesting issue here as well ?

Reply:
yeah it was suggesting here also

Reply:
we can go through slither warnings of this PR tomorrow while we discuss other slither warnings

Reply:
hmm cool